bazel/grpc_deps.bzl

```diff
@@ -195,9 +195,9 @@ def grpc_deps():
     if "com_google_absl" not in native.existing_rules():
         http_archive(
             name = "com_google_absl",
-            sha256 = "939e50c2fbcbd6f1124023350ef9bd5310d413227faf0e11de2a366dcc4e1e48",
-            strip_prefix = "abseil-cpp-a2e6adecc294dc4cd98cc285a9134ce58e0f2ad0",
-            url = "https://github.com/abseil/abseil-cpp/archive/a2e6adecc294dc4cd98cc285a9134ce58e0f2ad0.tar.gz",
+            sha256 = "979e3a5ff3f8ca417bc4350ebeadd7736c8e9f8ed5ac3d2c524074926c99ea44",
```

Comment:
It would be better to update to released version, but I see there is none.

Reply:
It'd be better to have a new released version here and Abseil team will have one this week.